annif/suggestion.py

```diff
@@ -136,11 +136,17 @@ def vector(self):
 
     def filter(self, limit=None, threshold=0.0):
         mask = (self._vector > threshold)
+        empty_labels = self._subject_index.empty_labels()
```

Comment:
Same comment as for the method name - something like `deprecated_ids` would be a better name for the variable